tests/js/cta-banners/banner-state-util.test.js

```diff
@@ -0,0 +1,12 @@
+const bannerStateUtils = require('../../../kuma/static/js/components/cta-banner/utils/banner-state-util.js');
+
+describe('setBannerState', function() {
+    it('throws and error if banner property is not set', function() {
+        function setBannerStateWithoutProp() {
+            bannerStateUtils.setBannerState();
+        }
+        expect(setBannerStateWithoutProp).toThrow(
+            'The banner property is required'
+        );
+    });
+});
```

Comment:
I would have expected a test here that verifies that setBannerState affects the output of isBannerActive. I don't know if jest includes built-in localStorage implementation or if you'd have to mock that...  I don't feel strongly about this, but maybe at least add a TODO comment indicating that such a test should be added in the future?

Reply:
Yup, I definitely intend to add additional tests.